pymagicc/io/binout.py

```diff
@@ -147,4 +139,118 @@ def process_data(self, stream, metadata):
             "todo": ["SET"] * len(regions),
```

Comment:
@znicholls Should the dat files be SET or N/A by default? The ASCII files are N/A and the old bin reader used SET.

Reply:
Hmm good question. I would say N/A and change the old bin reader